code/figure5.jl

```diff
@@ -0,0 +1,68 @@
+include("seed.jl")
+
+# Simulation with specialist natural enemies with stochasticity on D
+sim1, params1 = simulation(50.0, 25.0, m=0.2, F=4.0, D=0.5, c=1.0, a=0.5, th= 0.0, m=0.5, f=specialist_dyn, D_sd=0.5)
```

Comment:
Julia Base uses `std` for standard deviation. It would probably be good to keep the consistency.

Reply:
Done.